hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/rmadmin/TestFederationRMAdminInterceptor.java

```diff
@@ -22,6 +22,9 @@
 import org.apache.hadoop.metrics2.lib.DefaultMetricsSystem;
 import org.apache.hadoop.test.LambdaTestUtils;
 import org.apache.hadoop.yarn.api.records.DecommissionType;
+import org.apache.hadoop.yarn.api.records.NodeId;
+import org.apache.hadoop.yarn.api.records.Resource;
+import org.apache.hadoop.yarn.api.records.ResourceOption;
```

Comment:
Take a look at the checkstyle report, there are a few issues:
```
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/rmadmin/FederationRMAdminInterceptor.java:30:import org.apache.hadoop.yarn.api.records.NodeId;:8: Unused import - org.apache.hadoop.yarn.api.records.NodeId. [UnusedImports]
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/rmadmin/FederationRMAdminInterceptor.java:31:import org.apache.hadoop.yarn.api.records.ResourceOption;:8: Unused import - org.apache.hadoop.yarn.api.records.ResourceOption. [UnusedImports]
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/rmadmin/TestableFederationRMAdminInterceptor.java:27:import org.apache.hadoop.yarn.server.resourcemanager.MockNM;:8: Unused import - org.apache.hadoop.yarn.server.resourcemanager.MockNM. [UnusedImports]
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/rmadmin/TestableFederationRMAdminInterceptor.java:30:import org.junit.Assert;:8: Unused import - org.junit.Assert. [UnusedImports]
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/rmadmin/TestFederationRMAdminInterceptor.java:75:  private final int GB = 1024;:21: Name 'GB' must match pattern '^[a-z][a-zA-Z0-9]*$'. [MemberName]
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/rmadmin/TestFederationRMAdminInterceptor.java:351:    NodeId nodeId = NodeId.newInstance("127.0.0.1",1);:51: ',' is not followed by whitespace. [WhitespaceAfter]
```

Reply:
I will fix it.